src/merge.c

```diff
@@ -1711,8 +1732,13 @@ int git_merge__iterators(
 
 	git_vector_foreach(&changes, i, conflict) {
 		int resolved = 0;
+		int file_favor = opts.file_favor;
+
+		/* Check for merge options in .gitattributes */
+		if ((error = git_merge__lookup_file_favor(repo, conflict->our_entry.path, &file_favor) < 0))
```

Comment:
Right...  to be completely honest, it's probably easier to simply experiment.  I *suspect* that the ours side is the name to use, but I don't know for sure...  So they can definitely be different, when there's a rename conflict.  Eg, if you have an ancestor of `foo`, and the ours side renames `foo` -> `bar` and the theirs side renames `foo` -> `baz` then it's not clear to me which one you should use.

But also, there's no guarantee that there is an ours side (if it was deleted), so do be sure to test before you dereference it.